.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
+  - "gradle/"
+  - "**.gradle"
+BASH:
+  - "dev/check-license"
+  - "**.sh"
+DOCS:
+  - "site/"
+  - "!/site/mkdocs.yml"
+  - "**CHANGELOG.md"
+  - "**LICENSE"
+  - "**NOTICE"
+  - "**README.md"
+EXAMPLES:
+  - "examples/"
+COMMON:
+  - "common/"
+API:
+  - "api/"
+  - "python/iceberg/api/"
+  - "python/tests/api/"
+CORE:
+  - "**/core/"
+  - "core/"
+  - "python/iceberg/core/"
+  - "python/tests/core/"
+PARQUET:
+  - "**/parquet/"
+  - "parquet/"
+ARROW:
+  - "**/arrow/"
+  - "arrow/"
+ORC:
+  - "**/orc/"
+  - "orc/"
+HIVE:
+  - "**/hive/"
+  - "hive/"
+DATA:
+  - "**/data/"
+  - "data/"
+SPARK2:
```

Comment:
What about having a single Spark tag instead of 2? I think most of the time we will want to make sure that we are updating both versions.

Reply:
I think that's a good idea. I originally chose to follow just the modules as listed, but anybody interested in `SPARK` is likely interested in all of the spark related code.